src/draw-svg.ts

```diff
@@ -17,6 +18,7 @@ import {GeoElement} from './utilities';
 
 export type LineMark = 'bar'|'bar2'|'arrow'|'arrow2';
 export type LineArrow = 'start'|'end'|'both';
+const CIRCLE_MAGIC = 4*(Math.sqrt(2)-1)/3;
```

Comment:
Can you add a comment with a link to the source for this?

Reply:
done